test-signer/Dockerfile.amd64

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021-2023 Tigera, Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM scratch
+
+COPY --chown=10001:10001 ./bin/test-signer-amd64 /usr/bin/test-signer
```

Comment:
no need `--chown=10001:10001` and use `bin/test-signer-amd64` instead of `./bin`.